common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -0,0 +1,67 @@
+@import common.NewNavigation
+
+@import common.TopLevelSection
+
+@sectionList(topLevelSection: TopLevelSection) = {
+    <li class="main-navigation__item navigation-border">
+            <!-- TODO: Firefox -->
+        <details>
```

Comment:
No yet! That will have to happen in a separate PR. I will mention it in this PR's description.